# correspond to demo syntax result in https://cloud.google.com/natural-language
string name
string lemma
int32 dependency_edge
int32 part_of_speech
int32 parse_label
